src/Tabs/Tab.tsx

```diff
@@ -12,6 +12,7 @@ const StyledTab: React.FunctionComponent<TabProps> & { tabsRole: string } = prop
   return (
     <ClassNames>
       {({ css: getClassName }) => (
+        // @ts-ignore ref type is incompatible after @types/react bump
```

Comment:
The error here seems to be on the `css` property — isn't that related to Emotion rather than React?

In any case, I'd vote to revert the type bump; last thing I want to do is to put this statement in every component in this library.

Reply:
I think the error lies in LegacyRefObject, at least this is what TS complained about on my machine.